2021/10/21 05:02:08.158	error	http.log.access.log0	handled request	{"request": {"remote_addr": "192.168.31.31:8674", "proto": "HTTP/2.0", "method": "GET", "host": "192.168.31.9", "uri": "/favicon.ico", "headers": {"Sec-Ch-Ua": ["\"Chromium\";v=\"94\", \"Google Chrome\";v=\"94\", \";Not A Brand\";v=\"99\""], "Sec-Ch-Ua-Mobile": ["?0"], "User-Agent": ["Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.81 Safari/537.36"], "Sec-Ch-Ua-Platform": ["\"Windows\""], "Accept": ["image/avif,image/webp,image/apng,image/svg+xml,image/*,*/*;q=0.8"], "Sec-Fetch-Mode": ["no-cors"], "Referer": ["https://192.168.31.9/readme.html"], "Pragma": ["no-cache"], "Accept-Language": ["zh-CN,zh;q=0.9"], "Sec-Fetch-Site": ["same-origin"], "Sec-Fetch-Dest": ["image"], "Accept-Encoding": ["gzip, deflate, br"], "Cache-Control": ["no-cache"]}, "tls": {"resumed": false, "version": 772, "cipher_suite": 4865, "proto": "h2", "proto_mutual": true, "server_name": ""}}, "common_log": "192.168.31.31 - - [21/Oct/2021:05:02:08 +0000] \"GET /favicon.ico HTTP/2.0\" 404 0", "user_id": "", "duration": 0.000095213, "size": 0, "status": 404, "resp_headers": {"Server": ["Caddy"]}}
